docker/train.sh

```diff
@@ -30,13 +30,14 @@ echo ""
 --logdir /logs/train_log \
 --fp16 \
 --throughput_log_freq ${THROUGHPUT_LOG_FREQ} \
---config MODE_MASK=True \
+--config \
+MODE_MASK=True \
 MODE_FPN=True \
 DATA.BASEDIR=/data \
 DATA.TRAIN='["train2017"]' \
 DATA.VAL='("val2017",)' \
-TRAIN.STEPS_PER_EPOCH=${STEPS_PER_EPOCH} \
-TRAIN.LR_SCHEDULE='[120000, 160000, 180000]' \
+TRAIN.BATCH_SIZE_PER_GPU=1 \
+TRAIN.LR_EPOCH_SCHEDULE='[(8, 0.1), (10, 0.01), (12, None)]' \
```

Comment:
Can you make the corresponding changes in the ami script too?

Reply:
I made the changes. I really think we should abandon the AMI script - there is no advantage to not using Docker